analysis/port_analyzer.go

```diff
@@ -16,11 +24,8 @@ type PortAnalyzer struct {
 	period    sim.VTimeInSec
 	port      sim.Port
 
-	lastTime       sim.VTimeInSec
-	outTrafficByte uint64
-	outTrafficMsg  uint64
-	inTrafficByte  uint64
-	inTrafficMsg   uint64
+	lastTime          sim.VTimeInSec
+	PortAnalyzerTable map[string]PortAnalyzerEntry
```

Comment:
I think there is no need to make it a public field? Also, what is the key here? It is better to name it as `XxxToYyyMap`.